org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/DisplaySplash.java

```diff
@@ -0,0 +1,138 @@
+/*
+ *    Debrief - the Open Source Maritime Analysis Application
+ *    http://debrief.info
+ *
+ *    (C) 2000-2018, Deep Blue C Technology Ltd
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the Eclipse Public License v1.0
+ *    (http://www.eclipse.org/legal/epl-v10.html)
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ */
+package org.mwc.debrief.lite;
+
+/**
+ * @author Ayesha <ayesha.ma@gmail.com>
+ *
+ */
+import java.awt.AlphaComposite;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics2D;
+import java.awt.SplashScreen;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JFrame;
+import javax.swing.JProgressBar;
+import javax.swing.Timer;
+
+import org.mwc.debrief.lite.util.Utils;
+
+public class DisplaySplash extends JFrame implements ActionListener,Runnable {
+  private SplashScreen splash;
+  private int numTasks;
+  private static String debriefVersion;
+
+  static {
+    debriefVersion = Utils.readManifestVersion();
+  }
+  /**
+   * 
+   */
+  private static final long serialVersionUID = 1L;
+
+  static void renderVersion(Graphics2D g) {
+    g.setComposite(AlphaComposite.Clear);
+    g.fillRect(120,140,600,150);
+    g.setPaintMode();
+    g.setColor(Color.BLACK);
+    if(debriefVersion==null) {
+      debriefVersion="Dev Mode";
+    }
+    g.drawString(debriefVersion, 330, 170);
+  }
+  
+  static void renderSplashFrame(Graphics2D g, String message) {
+    g.setComposite(AlphaComposite.Clear);
+    g.fillRect(0,140,300,80);
+    g.setPaintMode();
+    g.setColor(Color.RED);
+    g.drawString(message, 20, 170);
+  }
+  public DisplaySplash(int numTasks) {
+    super();
+    this.numTasks=numTasks;
+
+    splash = SplashScreen.getSplashScreen();

```

Comment:
@ayeshare - I'm still unable to run this from the IDE.  This method returns null, so `splash` has null value.  We then get an NPE each time we refer to it.

Reply:
Here's how I passed the argument:
![image](https://user-images.githubusercontent.com/1108513/51971387-eba66b00-2470-11e9-969d-befa7d283ae1.png)